Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1351,41 +1351,32 @@ public function isMonolithicPhp()
             return false;
         }
 
-        $kinds = $this->findGivenKind(array(T_OPEN_TAG, T_OPEN_TAG_WITH_ECHO, T_INLINE_HTML));
-
-        /*
-         * Fix HHVM incompatibilities
-         */
-        $hhvmOpenTagsWithEcho = array();
-        $hhvmHashBangs = array();
-
-        if (defined('HHVM_VERSION')) {
-            /*
-             * HHVM parses '<?=' as T_ECHO instead of T_OPEN_TAG_WITH_ECHO
-             *
-             * @see https://github.com/facebook/hhvm/issues/4809
-             */
-            $hhvmEchoes = $this->findGivenKind(T_ECHO);
-            foreach ($hhvmEchoes as $token) {
-                if (0 === strpos($token->getContent(), '<?=')) {
-                    $hhvmOpenTagsWithEcho[] = $token;
-                }
+        $openTagsCounter = 0;
+
+        foreach ($this as $token) {
+            if ($token->isGivenKind(T_INLINE_HTML)) {
+                return false;
             }
 
-            /*
-             * HHVM parses "#!/usr/bin/env php\n" as T_HASHBANG (not defined in
```

Comment:
it's there: https://github.com/keradus/PHP-CS-Fixer/blob/8791b49cd6f714d0d0db19abf11485c862257f5e/Symfony/CS/Tests/Tokenizer/TokensTest.php#L870